docs/examples.rst

```diff
@@ -1,14 +1,10 @@
-STIPS Examples
-===============
-.. note::
+Additional STIPS Examples
+=========================
 
-    If you do not have environment variables pointing to the location of your Pandeia data,
-    STIPS data, and Webbpsf data, you must set these environment variables using `os.environ` or other
-    equivalent method prior to running any of these example scripts.
+Below is a brief condensed example of STIPS usage, similar to the usage in the :doc: `Basic Tutorial <basic_tutorial>`.
 
-
-* Creating a scene from an existing source catalogue `input_sources.txt`, and observing it with the
-  Roman WFI "F129" filter, offset by 0.5 degrees in RA, and rotated by 27 degrees:
+* Creating a scene from an existing source catalogue `input_sources.txt`, and observing it with the Roman WFI "F129" filter, offset by 0.5 
```

Comment:
```suggestion
* Creating a scene from an existing source catalog ``input_sources.txt``, and observing it with the Roman WFI F129 filter, offset by 0.5 
```
- You need two carats to get monospace text.
- The filter name doesn't need quotes.